libsolidity/codegen/CompilerContext.cpp

```diff
@@ -133,6 +133,53 @@ bool CompilerContext::appendCallback(evmasm::AssemblyItem const& _i) {
 	m_disable_rewrite = true;
 
 	auto callYUL = R"(
```

Comment:
This is basically a copy-paste of what was there before -- looks intimidating but the diff is just a few lines really